### DISCUSSION TITLE
Action never run

### DISCUSSION THREAD

**Discussion by KristjanVall** [community/community#49060] [Actions]
### Select Topic Area

Product Feedback

### Body

ID: C409:6A56:BF973F:C5F5FA:640135FA

is in Queued postion. And never run. Also it do not accept new actions tu run.

**Comment by ryush00** [comment]
We have same problem. We can't run self-hosted runner.

**Comment by seanmceligot** [comment]
Same here: https://github.com/seanmceligot/fastidious/actions #30.
I used "act" to fix my workflow locally but it won't run the new commit and the broken commit is Queued for 14 hours now. I get "Failed to cancel workflow. " error when I try to cancel it. Is this related to the Actions outage yesterday?

**Reply by seanmceligot** [reply]
I fixed mine by changing the Name: at the top of the workflow file.

**Comment by rungrotArunon** [comment]
Thank you brother for helping me fix the fever.

**Comment by jsoref** [comment]
You probably tripped on an outage, likely https://github.com/community/community/discussions/49104. The usual workaround for this is to create new commits (whether by rebasing, amending, or pushing additional commits) to the branch that triggered the workflow.

Yes, it's unfortunate, but software isn't guaranteed to run 💯% reliably 💯% of the time.
